hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/prehandle/PreHandleResult.java

```diff
@@ -37,6 +37,25 @@
  * Metadata collected when transactions are handled as part of "pre-handle". This is then made available to the
  * handle workflow.
  *
+ * <p>Several fields are pure functions of the input transaction. These are,
+ * <ul>
+ *     <li>{@link #payer}</li>
+ *     <li>{@link #txInfo}</li>
+ * </ul>
+ *
+ * <p>But the remaining fields depend on both the input transaction and the state of the world
+ * in which the transaction is being handled; both state and configuration. These are,
+ * <ul>
+ *     <li>{@link #payerKey}</li>
+ *     <li>{@link #status}</li>
+ *     <li>{@link #responseCode}</li>
+ *     <li>{@link #requiredKeys}</li>
+ *     <li>{@link #optionalKeys}</li>
+ *     <li>{@link #hollowAccounts}</li>
+ *     <li>{@link #verificationResults}</li>
+ *     <li>{@link #configVersion}</li>
+ * </ul>
```

Comment:
Is this related to the ISS you mentioned?  If so a note on why this information is important would be nice.

Reply:
Ah will elaborate in next PR.